stips/utilities/utilities.py

```diff
@@ -323,18 +323,18 @@ def DownloadReferenceData():
     else:
         print("Found at {}".format(synphot_data_path))
 
-    # webbpsf
-    print("Checking webbpsf data")
-    webbpsf_url = "https://stsci.box.com/shared/static/qxpiaxsjwo15ml6m4pkhtk36c9jgj70k.gz"
-    webbpsf_data_path = os.environ[GetParameter("webbpsf_data_name", use_data=False)]
-    webbpsf_data_file = "webbpsf_data.tar.gz"
-    if not os.path.isdir(webbpsf_data_path):
-        print("Downloading webbpsf data to {}".format(webbpsf_data_path))
-        os.makedirs(webbpsf_data_path)
-        get_compressed_file(webbpsf_url, webbpsf_data_file, webbpsf_data_path,
+    # stpsf
+    print("Checking stpsf data")
+    stpsf_url = "https://stsci.box.com/shared/static/qxpiaxsjwo15ml6m4pkhtk36c9jgj70k.gz"
+    stpsf_data_path = os.environ[GetParameter("stpsf_data_name", use_data=False)]
+    stpsf_data_file = "webbpsf_data.tar.gz"
+    if not os.path.isdir(stpsf_data_path):
+        print("Downloading stpsf data to {}".format(stpsf_data_path))
+        os.makedirs(stpsf_data_path)
+        get_compressed_file(stpsf_url, stpsf_data_file, stpsf_data_path,
```

Comment:
Replacing L326-L334 with the following should complete the name transition by downloading `stpsf-data-LATEST.tar.gz` from Box instead of `webbpsf-data-LATEST.tar.gz`:

```
    # stpsf
    stpsf_url = "https://stsci.box.com/s/kqfolg2bfzqc4mjkgmujo06d3iaymahv"
    stpsf_data_path = os.environ[GetParameter("stpsf_data_name", use_data=False)]
    stpsf_data_file = "stpsf_data.tar.gz"
    if not os.path.isdir(stpsf_data_path):
        print("Downloading stpsf data to {}".format(stpsf_data_path))
        os.makedirs(stpsf_data_path)
        get_compressed_file(stpsf_url, stpsf_data_file, stpsf_data_path,
                            "stpsf-data/")
```